lib/lita/handlers/jarvis.rb

```diff
@@ -21,11 +21,8 @@ class Jarvis < Handler
       fancy_route("cla", ::Jarvis::Command::CLA, :command => true, :flags => {
         "--cla-url" => ->(_) { config.find { |c| c.name == :cla_url }.value || raise(::Jarvis::Error, "Missing this setting in lita_config.rb: config.handlers.jarvis.cla_url") },
       })
-      fancy_route("publish", ::Jarvis::Command::Publish, :command => true, :flags => {
-        "--committer-email" => ->(request) { request.user.metadata["git-email"] || raise(::Jarvis::UserProfileError, "Missing user setting `git-email` for user #{request.user.name}") },
-        "--committer-name" => ->(request) { request.user.name },
-        "--github-token" => ->(_) { config.find { |c| c.name == :github_token }.value || raise(::Jarvis::Error, "Missing this setting in lita_config.rb: config.handlers.jarvis.github_token") }
```

Comment:
Not sure, I think the `publish_gem` uses `git push origin TAGS` and not the github api?